asv_bench/benchmarks/benchmarks.py

```diff
@@ -359,3 +359,48 @@ def setup(self, shape, columns_number, ascending_list):
 
     def time_sort_values(self, shape, columns_number, ascending_list):
         execute(self.df.sort_values(self.columns, ascending=self.ascending))
+
+
+class BaseReadCsv:
+    # test data file can de created only once
+    def setup_cache(self, test_filename="io_test_file"):
```

Comment:
We need a function that will delete the generated files after running the tests.

Reply:
Test file is created in the subdirectory in `/tmp` dicrectory (on Linux) and after benchmark finish this subdirectory automatically deletes.